### ISSUE TITLE
Directly add sub to multireddit

### ISSUE THREAD

**Issue by test2a** [ccrama/Slide#3288]
Slide version:
Android version:
Currently the only way to add subs to a multireddit is to select multiredit first, then pencil edit. Then search. Then save. Then save.

Why not when viewing a sub option to add to a multireddit ?

**Comment by tp0** [comment]
This is possible, it well hidden though:

When viewing s subreddit > view sidebar > tap the folder icon next to "subscribe".

**Comment by test2a** [comment]
yes. there is one "add to list" but....  its difficult. why not have a button"add to multireddit"  below subscribe because its independent to subscriptions

**Comment by tp0** [comment]
> yes. there is one "add to list" but.... its difficult. why not have a button"add to multireddit" below subscribe because its independent to subscriptions

No, this white folder icon. It is the same thing except that it is an icon instead of text. It is independent from subscribing.

![Screenshot_20201118-111041](https://user-images.githubusercontent.com/5863551/99509896-0ec84480-298f-11eb-8ba3-6176e100adc0.png)

**Comment by test2a** [comment]
OK. Got it. Thanks but there should be a label also.

**Comment by test2a** [comment]
Uh... where is the edit multireddit button ? I added a sub but now there is now nothing. The button next to gallery mode is now some image view. Cant find edit

**Comment by tp0** [comment]
> Uh... where is the edit multireddit button ? I added a sub but now there is now nothing. The button next to gallery mode is now some image view. Cant find edit

That's a sepate bug. #https://github.com/ccrama/Slide/issues/3294
